rust/docker-init.sh

```diff
@@ -10,23 +10,18 @@ if [[ "${ENABLE_MASQUERADE}" == "1" ]]; then
   ip6tables -t nat -A POSTROUTING -o eth+ -j MASQUERADE
 fi
 
-if [[ "${LISTEN_ADDRESS_DISCOVERY_METHOD}" == "gce_metadata" ]]; then
+if [ "${LISTEN_ADDRESS_DISCOVERY_METHOD}" = "gce_metadata" ]; then
   echo "Using GCE metadata to discover listen address"
 
-  if [[ "${PUBLIC_IP4_ADDR:-}" == "" ]]; then
+  if [ "${PUBLIC_IP4_ADDR:-}" == "" ]; then
     export PUBLIC_IP4_ADDR=$(curl "http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip" -H "Metadata-Flavor: Google" -s)
     echo "Discovered PUBLIC_IP4_ADDR: ${PUBLIC_IP4_ADDR}"
-  fi;
+  fi
 
-  if [[ "${PUBLIC_IP6_ADDR:-}" == "" ]]; then
+  if [ "${PUBLIC_IP6_ADDR:-}" == "" ]; then
     export PUBLIC_IP6_ADDR=$(curl "http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/ipv6s" -H "Metadata-Flavor: Google" -s)
     echo "Discovered PUBLIC_IP6_ADDR: ${PUBLIC_IP6_ADDR}"
-  fi;
+  fi
 fi
 
-# if first arg looks like a flag, assume we want to run postgres server
-if [ "${1:0:1}" = '-' ]; then
-  set -- ${PACKAGE_NAME} "$@"
-fi
```

Comment:
@AndrewDryga Was this still required for `rust` images?

Reply:
@jamilbk I think yes, @conectado knows why. On the onsite I was fixing this so just removed some hacks from bash but not 100% sure about this one.